src/actions/resize/ResizeSimpleAction.ts

```diff
@@ -8,20 +8,22 @@ import {ExpressionQualifier} from "../../qualifiers/expression/ExpressionQualifi
 
 /**
  * @description Defines a resize using width and height.
- * @extend {SDK.Action}
+ * @extends SDK.Action
+ * @memberOf Actions.Resize
+ * @see Visit {@link Actions.Resize| Resize} for examples
  */
 class ResizeSimpleAction extends Action {
   /**
    * @param {string} cropType
    * @param {number | string} cropWidth The required width of a transformed asset.
    * @param {number | string} cropHeight The required height of a transformed asset.
    */
-  constructor(cropType: string, cropWidth: number|string, cropHeight?: number|string) {
+  constructor(cropType: string, cropWidth: number | string, cropHeight?: number | string) {
     super();
-    if(cropWidth) {
+    if (cropWidth) {
```

Comment:
Do you like having the ifs with a space like that?

Reply:
Yes, I do! 

Most of the project is written like that as well!